docs/manage/data/export.md

```diff
@@ -43,4 +43,4 @@ You can see more information about exporting data in the [Viam CLI documentation
 
 For a comprehensive tutorial on data management, see [Intro to Data Management](../../../tutorials/services/data-management-tutorial).
 
-If you have collected image data, you can use that data to [train image classification model](../train-model) within Viam.
+If you have collected image data, you can use that data to [train image classification model](../../ml/train-model) within Viam.
```

Comment:
does this really belong here?  If they have exported data, they could use their own ML training mechanisms and then import the model - is this what we are conveying?  If so, by the way they can upload detector models as well since we support those in our vision service